doc/basics/compilation.md

```diff
@@ -116,6 +116,8 @@ RADIUM_UPDATE_VERSION:BOOL=ON
 
 All radium related cmake options (with their current values) can be printed with `cmake -LAH | grep -B1 RADIUM` (on linux like system)
 
+\warning For computers with low RAM capacities (under 12G) we recommend to set the `CMAKE_BUILD_PARALLEL_LEVEL` environment variable to a reasonable value (i.e. 2) to prevent the computer from swapping.
```

Comment:
```suggestion
\warning For computers with low RAM capacities (under 12G) we recommend to set the `CMAKE_BUILD_PARALLEL_LEVEL` variable to a reasonable value (i.e. 2) to prevent the computer from swapping.
```

Reply:
The variable has to be set in the environment, setting it with `-DCMAKE_BUILD_PARALLEL_LEVEL` does not work since it is fetched with `ENV{CMAKE_BUILD_PARALLEL_LEVEL}`. I would rather not remove that precision.

Reply:
**TL;DR** : The choice of using ENV variable is explained in the issue #611 fixed by the PR #612.

As said in the doc, the `CMAKE_BUILD_PARALLEL_LEVEL` initial value is taken from the calling process environment
(https://cmake.org/cmake/help/v3.18/envvar/CMAKE_BUILD_PARALLEL_LEVEL.html) and, if this variable is not defined in the environment, the parallel build will use the default value of the native build tool.

These lines allows to change the default build tool parallel build level while still allowing the user to control this value.

Most of the native build tool have a default value to 1, that makes the compilation very long. But setting the value to the number of core whithout testing the ENV variable prevent to build and install the libs on low memory computer (e.g. some CI docker runners only offers 4 Gb) or prevent to do a `build -j xxx`, as the build tools does not accept that one overrides the jobs number.